src/constants.ts

```diff
@@ -25,6 +25,7 @@ export function getDefaultCPUs(preset: Preset): number {
     case 'openshift':
       return 4;
     case 'microshift':
+      return 2;
```

Comment:
?

Reply:
That is default/minimum values, we use them to validate user changes in a preferences